src/wallet.js

```diff
@@ -918,7 +918,7 @@ const wallet = {
 
       newAccessData = {
         ...newAccessData,
-        acctPathMainKey: encryptedAccPathMainKey,
+        acctPathMainKey: encryptedAccPathMainKey.encrypted.toString(),
```

Comment:
Is this a bug we found on this PR?

Reply:
Yes it is, that's the bug I described on the PR description:

`It also contains a fix for the changePin method`